resolwe/flow/views/entity.py

```diff
@@ -48,6 +48,19 @@ def _check_collection_permissions(self, collection_id, user):
             else:
                 raise exceptions.NotFound()
 
+        return collection
```

Comment:
This is a bad idea to return the collection. If you want to do this, you have to rename the function to `_get_collection_for_user` (in a separate commit, of course).

Reply:
This still needs to be fixed.

Reply:
I see this was fixed in the next commit. Switch commits, to make it nicer and move this line to the other commit.